lib/ae_mdw/db/mutations/aex9_account_balance_mutation.ex

```diff
@@ -0,0 +1,80 @@
+defmodule AeMdw.Db.Aex9AccountBalanceMutation do
```

Comment:
Separate mutation as the ContractCallMutation is still on Mnesia. During the migration to RocksDb we will be able increase the cohesion if needed.